Config/Tests iOS.xcconfig

```diff
@@ -1,9 +1,11 @@
 #include "./Tests.xcconfig"
 #include "./iOS.xcconfig"
 
-OTHER_LDFLAGS = $(inherited) -framework UIKit -ObjC -framework CrashReporter -framework SafariServices -framework WebKit -framework AuthenticationServices -lz
+OTHER_LDFLAGS = $(inherited) -framework UIKit -framework SafariServices -framework WebKit -framework AuthenticationServices -lz
 
 LD_RUNPATH_SEARCH_PATHS = @executable_path/Frameworks @loader_path/Frameworks "$(SRCROOT)/../Vendor/iOS/OCMock" "$(SRCROOT)/../Vendor/iOS/OCHamcrest" "$(SRCROOT)/../Vendor/iOS/OHHTTPStubs"
 
 // OCMock and OCHamcrest have a bunch of warnings so we just disable this only for testing.
 GCC_TREAT_WARNINGS_AS_ERRORS = NO
+
+USER_HEADER_SEARCH_PATHS=$(inherited) $(SRCROOT)/MSAL/**
```

Comment:
Isn't that just for Identity? Can probably be moved to dedicated test config?

Reply:
Oops. Yeah.. I didn't realize that it is a common one. I have to revert one change that I made above and capture a work item for this one to separate.